build/lib/fetch.ts

```diff
@@ -58,7 +58,10 @@ export async function fetchUrl(url: string, options: IFetchOptions, retries = 10
 			startTime = new Date().getTime();
 		}
 		const controller = new AbortController();
-		const timeout = setTimeout(() => controller.abort(), 30 * 1000);
+		// --- Start Positron ---
+		// Increase the timeout to 90 seconds to accommodate slower downloads.
+		const timeout = setTimeout(() => controller.abort(), 90 * 1000);
```

Comment:
This is the simplest fix, but we could be retrying 10 times at 90 seconds each. Should we instead increase the timeout at each retry?

Reply:
If we're worried, quarto.ts could call fetchUrl() directly since it's a list of a single asset. Or, we could update IFetchOptions so any caller to fetchUrls() could specify their own custom retries or timeout.

Reply:
I've changed the timeout back to a default of 30s, but added `timeoutSeconds` to `IFetchOptions` and pass 90s as the timeout from `quarto.ts`. So the existing behaviour should remain the same for the other downloads.

I didn't change quarto.ts to call `fetchUrl()` to avoid having to re-Stream-ify the files.

Let me know what you think!